velox/functions/prestosql/registration/JsonFunctionsRegistration.cpp

```diff
@@ -39,4 +40,42 @@ void registerJsonFunctions() {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_json_parse, "json_parse");
 }
 
+void registerSIMDJsonFunctions() {
+  registerType("json", std::make_unique<const JsonTypeFactories>());
+
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Json, bool>(
+      {"simd_json_array_contains"});
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Json, int64_t>(
+      {"simd_json_array_contains"});
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Json, double>(
+      {"simd_json_array_contains"});
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Json, Varchar>(
+      {"simd_json_array_contains"});
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Varchar, bool>(
+      {"simd_json_array_contains"});
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Varchar, int64_t>(
+      {"simd_json_array_contains"});
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Varchar, double>(
+      {"simd_json_array_contains"});
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Varchar, Varchar>(
+      {"simd_json_array_contains"});
+
+  registerFunction<SIMDJsonParseFunction, Varchar, Varchar>(
+      {"simd_json_parse"});
+  registerFunction<SIMDJsonExtractScalarFunction, Varchar, Varchar, Varchar>(
+      {"simd_json_extract_scalar"});
+  registerFunction<SIMDJsonValidFunction, int64_t, Varchar>(
+      {"simd_json_valid"});
```

Comment:
json_valid and json_keys do not appear to be Presto functions: https://prestodb.io/docs/current/functions/json.html

Let's remove.